.svlint.toml

```
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+[option]
+exclude_paths = ["build.*", "sw/.*", ".sv.tpl$"]
+
+[rules]
+case_default = true
+enum_with_type = true
+for_with_begin = true
+function_same_as_system_function = false
+function_with_automatic = true
+generate_for_with_label = false
+generate_if_with_label = false
+generate_keyword_forbidden = true
+generate_keyword_required = false
+genvar_declaration_in_loop = true
+genvar_declaration_out_loop = false
+if_with_begin = false
+inout_with_tri = false
+input_with_var = false
+interface_port_with_modport = false
+legacy_always = false
+level_sensitive_always = true
+loop_variable_declaration = true
+non_ansi_module = true
+output_with_var = false
+parameter_in_package = false
+priority_keyword = true
+tab_character = true
+unique0_keyword = false
+unique_keyword = false
+wire_reg = false
```

```suggestion
wire_reg = true
```

we require logic over wire. (I'm not sure about the "polarity" of this flag, the docs at https://github.com/dalance/svlint/blob/master/RULES.md#wire_reg aren't quite clear to me.